resources/web/docs_js/components/console_widget.jsx

```diff
@@ -69,23 +69,80 @@ export const ConsoleForm = connect((state, props) =>
         `${props.setting}_curl_password`], state.settings)
 , {saveSettings})(_ConsoleForm);
 
+const alternativePrettyName = rawName => {
+  switch(rawName) {
+    case 'console': return 'Console';
+    case 'csharp': return 'C#';
+    case 'js': return 'JavaScript';
+    case 'php': return 'PHP';
+    default: return rawName;
+  }
+};
+
+const alternativeChoice = rawName => {
+  return <option value={rawName}>{alternativePrettyName(rawName)}</option>;
+};
+
+const alternativePicker = props => {
+  if (!props.alternatives) {
+    return;
+  }
+  const consoleAlternatives = props.alternatives.console;
+  if (!consoleAlternatives) {
+    return;
+  }
+
+  const items = [];
+  let sawChoice = 'console' === props.consoleAlternative;
+  items.push(alternativeChoice('console'));
+  for (const name of Object.keys(consoleAlternatives)) {
+    sawChoice |= name === props.consoleAlternative;
+    items.push(alternativeChoice(name));
```

Comment:
`items.push(<AlternativeChoice name={name} />)` and in the function grab the name off props